### ISSUE TITLE
Failed to check DTR bootstrapper image version

### ISSUE THREAD

**Issue by 53d117460ec63d70** [Mirantis/launchpad_legacy#54]
```
$ launchpad version
version: 1.0.2
commit: ad58f68
```
We are seeing the following FATAL error when deploying DTR without internet access:
```
INFO[0856] ==> Running phase: Upgrade DTR components
DEBU[0856] host.domain.com: found DTR installed, using as leader
DEBU[0856] analytics disabled, not tracking event 'Upgrade DTR components'
INFO[0856] See /home/deploy/.mirantis-launchpad/cluster/launchpad-de/apply.log for more logs                                                                                     DEBU[0856] analytics disabled, not tracking event 'Cluster Apply Failed'
FATA[0856] Failed to check DTR bootstrapper image version
```
I notice in this release the option ```--disable-upgrade-check``` is missing which is available in ```1.1.0-beta5```, could this option enable us to work around the issue?

**Comment by kke** [comment]
> I notice in this release the option --disable-upgrade-check is missing which is available in 1.1.0-beta5, could this option enable us to work around the issue?

No, that's just for disabling the launchpad's self-upgrade check.

It seems the check is using `<dtr.imageRepo>/dtr` which is wrong when using a custom image repo, as the image has been retagged to either `docker.io/docker/dtr` or `Mirantis/dtr`.

Fixing this asap.

**Comment by kke** [comment]
Fixed in https://github.com/Mirantis/launchpad/releases/tag/1.0.3
